frontend/main.js

```diff
@@ -148,10 +148,10 @@ async function startApp () {
         }, 8e3)
       })]
   ).catch(e => {
-    console.error('[main] Error setting up service worker', e)
-    alert(L('Error while setting up service worker'))
-    window.location.reload() // try again, sometimes it fixes it
-    throw e
+    // console.error('[main] Error setting up service worker', e)
+    // alert(L('Error while setting up service worker'))
+    // window.location.reload() // try again, sometimes it fixes it
+    // throw e
```

Comment:
Why have these been commented out?

Reply:
forgot to uncomment it again after development.

I commented out because this block leads to `reload -> error -> reload ...` infinite cycle, when using below link on my phone for investigation.

<img src='https://github.com/user-attachments/assets/4a7fcf25-076c-4686-b1f5-801f0ecaaa96' width='480'>